.github/workflows/test.yml

```
@@ -42,6 +39,42 @@ jobs:
           name: package
           path: ./package.tgz
 
+  test-dist:
+    name: Test against dist
+    needs: [build]
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node: ['20.x']
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Use node ${{ matrix.node }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node }}
+          cache: 'yarn'
+
+      - name: Install deps
+        run: yarn install
+
+      - uses: actions/download-artifact@v3
```

```suggestion
      - uses: actions/download-artifact@v4
```
Needs to be v4 to find the upload

---

Yes sirrrr